### PR TITLE
Add UampBrowseScreen

### PR DESCRIPTION
**Pull request by luizgrp**
#### WHAT

Add `UampBrowseScreen`.

Screenshot_20220729_114338

#### WHY

In order to display the list of downloaded playlists.

#### HOW

- Add `UampBrowseScreen` and `UampBrowseScreenViewModel`

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: media-sample/src/main/java/com/google/android/horologist/mediasample/ui/mapper/DownloadMediaUiModelMapper.kt [hunk 1]
```diff
@@ -36,10 +36,12 @@ object DownloadMediaUiModelMapper {
...
 
     fun mapList(mediaDownloadList: List<MediaDownload>): List<DownloadMediaUiModel> =
-        mediaDownloadList.map(::map)
+        mediaDownloadList.map(DownloadMediaUiModelMapper::map)
 
     fun map(media: Media): DownloadMediaUiModel =
         DownloadMediaUiModel.Unavailable(MediaUiModelMapper.map(media))
 
-    fun map(mediaList: List<Media>): List<DownloadMediaUiModel> = mediaList.map(::map)
+    fun map(mediaList: List<Media>): List<DownloadMediaUiModel> = mediaList.map(
```
Comment (luizgrp) [inline media-sample/src/main/java/com/google/android/horologist/mediasample/ui/mapper/DownloadMediaUiModelMapper.kt:44]: delete these 3 map functions as they are not in use

**Review by yschimke — APPROVED** [review]
Nice! Are the tile images meant to be included.

**Comment by yschimke** [comment]
Your PR has tile images included https://github.com/google/horologist/pull/447/files#diff-f3c37d8c24d7ffe674358e718db53b29ccc272e1e296cd113d34b4bb66723481

Thought these were in the other PR, so confused if this was reverting?

File: media-sample/src/main/java/com/google/android/horologist/mediasample/ui/mapper/DownloadMediaUiModelMapper.kt (hunk 1 above)
Comment (luizgrp) [reply media-sample/src/main/java/com/google/android/horologist/mediasample/ui/mapper/DownloadMediaUiModelMapper.kt:44]: https://github.com/google/horologist/pull/449

**Comment by luizgrp** [comment]
@yschimke oh I missed that, thanks, yeah not sure why it was picking up the difference, given it was supposed to be fixed in https://github.com/google/horologist/pull/448, will monitor 👍

**Pull request merged**